test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -793,7 +814,44 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 		framework.ExpectNoError(waitTillAllNAPNodePoolsAreRemoved())
 		By("Check if NAP group was removeed")
 		Expect(getNAPNodePoolsNumber()).Should(Equal(0))
-		framework.ExpectNoError(disableAutoprovisioning())
+	})
+
+	It("shouldn't add new node group if not needed [Feature:ClusterSizeAutoscalingScaleWithNAP]", func() {
+		framework.SkipUnlessProviderIs("gke")
+		const extraPoolName = "extra-pool"
+		addNodePool(extraPoolName, "n1-standard-4", 0)
```

Comment:
Is autoscaling enabled for this node group? If not it doesn't exactly serve it purpose:)

Reply:
Done